actions.md

```diff
@@ -0,0 +1,547 @@
+
+If an action exceeds its configured time limit, [an error is recorded in the activation record](#the-activation-record).
+
+#### Asynchronous requests
+
+In contrast, when the activation request is asynchronous, the HTTP request terminates once the system has accepted the request to invoke an action. With an asynchronous request, the system returns an activation ID to confirm that the invocation was received. You can use this ID to retrieve the activation record.
+
+To run a `nim` command asynchronously, use the `--no-wait` parameter, or `-n` for short, as in this command to invoke the `greeting` action:
+
+ ```
+ nim action invoke greeting --no-wait
+```
+
+**Tip:** If you're an OpenWhisk developer, you'll notice that the `wsk action invoke` is asynchronous by default, wherease the `nim action invoke` command is synchronous by default.
+
+### The activation record
+
+Each invocation of an action results in an activation record, which contains the following fields:
+
+- `activationId`: The activation ID.
+- `namespace` and `name`: The namespace and name of the entity.
+- `start` and `end`: Timestamps recording the start and end of the activation. The values are in [UNIX time format](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap04.html#tag_04_15).
+- `logs`: An array of strings with the logs that are produced by the action during its activation. Each array element corresponds to a line output to `stdout` or `stderr` by the action, and includes the time and stream of the log output. The structure is as follows: `TIMESTAMP` `STREAM:` `LOG LINE`.
+- `annotations`: An array of key-value pairs that record [metadata](annotations.md#annotations-specific-to-activations) about the action activation.
+- `response`: A dictionary that defines the following keys
+  - `status`: The activation result, which might be one of the following values:
+    - *"success"*: the action invocation completed successfully.
+    - *"application error"*: the action was invoked, but returned an error value on purpose, for instance because a precondition on the arguments was not met.
+    - *"action developer error"*: the action was invoked, but it completed abnormally, for instance the action did not detect an exception, or a syntax error existed. This status code is also returned under specific conditions such as:
+      - the action failed to initialize for any reason
+      - the action exceeded its time limit during the init or run phase
+      - the action specified a wrong docker container name
+      - the action did not properly implement the expected [runtime protocol](actions-new.md)
+    - *"whisk internal error"*: the system was unable to invoke the action. **[[NH: is 'whisk' ok here? Note also in table below]]**
```

Comment:
I think it's a question of what text our system actually uses (the word 'whisk' is not poisonous in itself).  @rabbah should comment.